### DISCUSSION TITLE
Cannot select object any more by left-mouse click!

### DISCUSSION THREAD

**Discussion by GitHubbet** [Stellarium/stellarium#1698] [Q&A] [answered]
Hi
Cannot select object any more by left-mouse click - only works if I search for it [F3, etc.]

I had last 20v version, then updated to latest now and it is just the same.

Was working last time I used it ... maybe a week ago.

Any ideas?

Thanks
M

**Comment by GitHubbet** [comment, marked as answer]
🤣😂 idiot me. I re-run it and found that CTRL+A was activated.
Switched off and it now works.

I thought that must work if I 'only' do click on object ... i.e. with CTRL+A on!
If you move, then CTRL+A enters in action, right?

**Reply by alex-w** [reply]
Angle Measure plugin was enabled by Ctrl+A shortcuts ;)

**Comment by axd1967** [comment]
You are not idiot, I've also been bitten several times by this behaviour. It's a small example of ergonomy, and there is a solution.

https://github.com/Stellarium/stellarium/blob/450127b543a83e52bdc96184dccee17be930d221/plugins/AngleMeasure/README#L9

A small improvement is to [change the cursor during angle measuring](https://doc.qt.io/archives/qt-4.8/qapplication.html#setOverrideCursor). I'll give it a shot.

**Reply by GitHubbet** [reply]
Thank you ... glad I mentioned it and it is easily fixable, as you said.

I am no developer, just a quite advanced user that dabs [bits and pieces] on PHP [used to!] HTML CSS and played a bit with Python - just only reached the point I managed to make a mini calculator.

Too 'old' to  learn now.. But I am grateful for the effort in keeping Stellarium in tip-top form.

In my older age, now I am more of a grumpy user!
hahahaha

**Comment by gzotti** [comment]
Indeed the angle measuring persists a program exit and relaunch. (There is a flag in the config.ini which is named accordingly.) Measuring should be disabled on program exit, or, to keep current functionality for I don't know who, we can add another config option.

**Reply by alex-w** [reply]
Why? Just remove support current config option to avoid saving the state.

**Reply by alex-w** [reply]
> Why? Just remove support current config option to avoid saving the state.

Done.

**Reply by axd1967** [reply]
indeed not worth saving the state.

**Reply by gzotti** [reply]
Fine with me. Did you remove enable_at_start? Or can whoever added it still write it manually?

**Reply by alex-w** [reply]
> Fine with me. Did you remove enable_at_start? Or can whoever added it still write it manually?

I did removed the support of this option whatever.
